docs/conf.py

```diff
@@ -0,0 +1,14 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+from rocm_docs import ROCmDocs
+
+docs_core = ROCmDocs("hipBLAS Documentation")
```

Comment:
hipBLASLt Documentation

Reply:
Thanks for catching that. Fixed in [56f2843](https://github.com/ROCmSoftwarePlatform/hipBLASLt/pull/64/commits/56f2843dbfea966ea2d66c3e7d9cc6482e8db389)